### PR TITLE
Homepage

### LINKED ISSUE #2: As a user, when I visit the app, all movies should be displayed
- Based on the design selection, the user should see the homepage with the right navbar and main section.
- The main section is where the movies will be displayed.
- Use the dummy data for movies.

Scenario: A user visits the app
Acceptance criteria: All the movies from the dummy data should be displayed

### PR DESCRIPTION
**Pull request by paulina-isabel**
This PR will close #2 

#### What is the feature?
- A main homepage view, displaying all movie cards 
- Each card contains the movie poster and the movie title

#### How you implemented the solution?
- Made four new components in their own files accounting for the different parts of the project's main page: App, Movies, Card, and Navbar. Added simple styling with css to each file.

#### Any changes in the UI (Screenshots)?
Screenshot 2023-07-06 at 7 50 54 PM

#### How to test the feature (A test scenario or any new setup)?
- No testing required yet.

#### Any other notes:
- Need to add a few changes in styling and the main element contents of the navbar.

### CODE REVIEW

File: src/Card.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import './Card.css';
+
+function Card({ movieTitle, moviePoster }){
```
Comment (taranveersingh93) [inline src/Card.js]: nice use of props

File: src/Card.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  return (
+    <div className='movie-card'>
+      <img src={moviePoster} className='movie-poster' alt={movieTitle} />
```
Comment (taranveersingh93) [inline src/Card.js]: thanks for having the alt text. I usually miss that

File: src/Movies.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import './Movies.css';
+import Card from './Card';
+
+function Movies({ movies }){
```
Comment (taranveersingh93) [inline src/Movies.js]: do we want ES5 functions or ES6 functions? (function keyword or arrow functions)?

File: src/Navbar.css [hunk 4]
```diff
@@ -0,0 +1,11 @@
+.navbar {
+  height: 20vh;
```
Comment (taranveersingh93) [inline src/Navbar.css]: thanks for using dynamic unit (vh)

**Review by taranveersingh93 — CHANGES_REQUESTED** [review]
Thank you! Just a couple of comments

File: src/Movies.js (hunk 3 above)
Comment (paulina-isabel) [reply src/Movies.js]: ES6 sounds good to me, actually. I made the changes and pushed!

**Review by taranveersingh93 — APPROVED** [review]
Nicely done! I'll take it up from here

**Pull request merged**